ACMILabs/interactive-label
# Cabinet wireframe

Replaces the front end with the SVGs-on-background-image approach.

To test:
```
$ cd development
$ docker-compose up --build
```
- Open chrome to http://localhost:8081
- Click something with a pink outline
- Check that the label appears, and closes when clicking the background
- Check that the chrome console has logs about setting and unsetting the current label (alt-cmd-j).

By the way, there was lightbox functionality for the images, but it has been turned off until we've locked something in. Please don't worry about all the commented out @LightboxesAreOff bits
